### PR TITLE
clear input when select || click out input

### LINKED ISSUE #38: clear input after search

### PR DESCRIPTION
**Pull request by husamkamal**
Relates #38

### CODE REVIEW

**Review by sara219 — DISMISSED** [review]

File: js/home.js [hunk 1]
```diff
@@ -51,6 +52,8 @@ inputField.addEventListener("keypress", (e) => {
         searchTerm = "Palestine";
       fetchWeatherData(searchTerm);
     }
+    inputField.value=""
+    suggBox.textContent=""
```
Comment (zaher-aa) [inline js/home.js:56]: Add spaces between the assign symbol and other stuff 😅
`inputField.value = ""`
`suggBox.textContent = ""`

File: js/home.js [hunk 2]
```diff
@@ -129,6 +132,7 @@ fetch("https://countriesnow.space/api/v0.1/countries/population/cities")
       inputBox.value = selectUserData;
       fetchWeatherData(inputBox.value);
       suggBox.textContent = "";
+      inputBox.value=""
```
Comment (zaher-aa) [inline js/home.js:135]: here too

File: js/home.js [hunk 3]
```diff
@@ -146,6 +150,7 @@ fetch("https://countriesnow.space/api/v0.1/countries/population/cities")
     document.body.onclick = (e) => {
       if (!e.target.matches(".recommends span, form input")) {
         suggBox.textContent = "";
+        inputBox.value=""
```
Comment (zaher-aa) [inline js/home.js:153]: and here

**Review by zaher-aa — CHANGES_REQUESTED** [review]

File: js/home.js (hunk 3 above)
Comment (zaher-aa) [inline js/home.js]: Why remove the input value when clicking outside the input field

File: js/home.js (hunk 2 above)
Comment (zaher-aa) [inline js/home.js]: Why remove the input value when selecting a place

File: js/home.js (hunk 1 above)
Comment (zaher-aa) [inline js/home.js]: Why remove the input value when searching for a place

**Review by zaher-aa — CHANGES_REQUESTED** [review]

**Review by zaher-aa — APPROVED** [review]

**Review by zaher-aa — APPROVED** [review]

**Review by AseelL — APPROVED** [review]

**Review by sara219 — APPROVED** [review]

**Pull request merged**